packages/react-core/src/components/DataList/DataListItem.tsx

```diff
@@ -82,7 +82,7 @@ export class DataListItem extends React.Component<DataListItemProps> {
               className={css(
                 styles.dataListItem,
                 isExpanded && styles.modifiers.expanded,
-                isSelectable && styles.modifiers.selectable,
```

Comment:
Does this correlate to an issue?  If so is there still additional work needed?

Reply:
This does not correlate with an issue - the core issue was marked as needing no react follow up: https://github.com/patternfly/patternfly/issues/5347
@thatblindgeye do you know if there is other follow up work needed for this update?